data/endpoints/fps-initiate-payment-v3.json

```diff
@@ -1,777 +1,816 @@
+                      "$ref": "#/components/schemas/BatchAmount",
+                      "description": "Indicates the amount and the currency used in a given transaction or account balance."
+                  },
+                  "creditor": {
+                      "$ref": "#/components/schemas/BatchCreditorPartyIdentifier",
+                      "description": "The name and, optionally, the legal entity identifier of the account."
+                  },
+                  "creditorAccount": {
+                      "$ref": "#/components/schemas/BatchPaymentInstructionCounterpartAccount",
+                      "description": "Information about the counterpart in a given transaction."
+                  },
+                  "remittanceInformation": {
+                      "$ref": "#/components/schemas/BatchRemittanceInformation",
+                      "description": "Information supplied to enable the matching/reconciliation of an entry with the items that the payment is intended to settle, such as commercial invoices in an accounts’ receivable system."
+                  }
+              },
+              "additionalProperties": false,
+              "description": "Represents a creditor in a payment instruction."
+          },
+          "BatchCreditTransferResponse": {
+              "type": "object",
+              "properties": {
+                  "transactions": {
+                      "type": "array",
+                      "items": {
+                          "$ref": "#/components/schemas/PaymentInstructionResponse"
+                      },
+                      "description": "The acceptance or rejection status of each individual payment instruction",
+                      "nullable": true
+                  },
+                  "halLinks": {
+                      "type": "array",
+                      "items": {
+                          "$ref": "#/components/schemas/HalLink"
+                      },
+                      "description": "Hypermedia links.",
+                      "nullable": true
+                  }
+              },
+              "additionalProperties": false,
+              "description": "The result of each payment instruction and applicable links"
+          },
+          "BatchCreditorPartyIdentifier": {
+              "required": [
+                  "name"
+              ],
+              "type": "object",
+              "properties": {
+                  "name": {
+                      "maxLength": 140,
+                      "minLength": 1,
+                      "type": "string",
+                      "description": "The name of the account holder."
+                  },
+                  "legalEntityIndentifier": {
+                      "maxLength": 35,
+                      "minLength": 0,
+                      "type": "string",
+                      "description": "The legal entity identifier of the account holder (eg: charity number). This should be supplied if known.",

```

Comment:
Do we actually mean the LEI, as defined in the LEI Register (https://www.legalentityidentifier.co.uk/)?
If so, it should probably be capitalised, Legal Entity Identifier, throughout.

Reply:
```suggestion
                      "description": "The Legal Entity Identifier of the account holder (eg: charity number). This should be supplied if known.",

```